src/components/CVModal/container.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { fetchCV } from 'Services';
+import { getEmployeeCV } from 'Reducers/selected-options';
+
+const CVModalContainer = Wrapped =>
+  class extends React.Component {
+    static propTypes = {
+      employeeCV: PT.shape({ id: PT.number, label: PT.string }).isRequired,
+    };
+
+    state = { cv: null };
+
+    componentDidMount() {
+      fetchCV().then(data => this.setState({ isModalOpen: false, cv: data }));
```

Comment:
As commented above, not a big fan of cv as a variable. Can we more detailed? I can't think of the top of my head though.....hmmm.

Reply:
I have changed the var name to cvURL